VERSION

```diff
@@ -1,14 +1,6 @@
 MYSQL_VERSION_MAJOR=5
 MYSQL_VERSION_MINOR=7
 <<<<<<< HEAD
-MYSQL_VERSION_PATCH=30
-MYSQL_VERSION_EXTRA=-33
-PXC_VERSION_EXTRA=43
```

Comment:
What does PXC_VERSION_EXTRA mean? It was 57 originally (indicating 5.7 ver of server?) then changed to 43 (what for?)

Reply:
It relaced WSREP_INTERFACE_VERSION, and based on the use, it should be the last part of the version string, which is currently 43.